examples/tomo/astra_parallel_2d_fp.py

```diff
@@ -1,67 +0,0 @@
-# Copyright 2014, 2015 The ODL development group
```

Comment:
OK. The xray example shouldn't be doing all the cases at once then, but that's out of scope here.